lib/ansible/plugins/inventory/toml.py

```diff
@@ -0,0 +1,252 @@
+# Copyright (c) 2018 Matt Martz <matt@sivel.net>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
```

Comment:
not core?

Reply:
There has been some discussion about this. We didn't want to indicate that it was fully supported. Based on my last set of conversations, it will be core, but stay preview.

Additionally, when using the toml format, warnings about it's preview status will be emitted.